main/zoe/guide/contracts/callSpread.md

```diff
@@ -0,0 +1,77 @@
+# Call Spread
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/callSpread.js) (Last updated: 03-NOV-2020)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+This contract implements a fully collateralized call spread. A call spread can be used as a
+financial building block to create futures, puts, calls, and event binaries that would form the
+basis for a prediction market, insurance, and much more. A Call Spread is a combination of a call
+option bought at one strike price and a second call option sold at a higher price. That means there
+are two parties to the call spread, each holding an ERTP asset that pays out complementary amounts
+depending on the value of some good at a known future time.
+
+The Zoe invitations representing the options are produced in pairs that represent the matching
+assets.  The purchaser deposits the entire amount that will be approtioned between the holders of
+the two positions. The individual options are ERTP invitations that are suitable for resale.
+
+This option is settled financially. There is no requirement that the original purchaser have
+ownership of the underlying asset at the start, and the beneficiaries shouldn't expect to take
+delivery at closing.
+
+It is common for the strike price and collateral to use the same currency, though this contract
+decouples them. This would be like a spread based on the price of APPL stock (`Underlying`),
+where the stock price is measured in USD (`Strike`), but the contract pays out in JPY
+(`Collateral`).
+
+The issuerKeywordRecord specifies the issuers for four keywords: Underlying, Strike, Collateral,
+and Quote. The original deposit and the payout are in Collateral. Strike amounts are used for the
+price oracle's quote as to the value of the Underlying, as well as the strike prices in the
+terms. The priceAuthority's quotes are ERTP assets issued by a QuoteAuthority, which must be
+specified using the keyword Quote.
```

Comment:
```suggestion
The issuerKeywordRecord specifies the issuers for four keywords: 
- Underlying: Strike amounts are used for the price oracle's quote.
- Strike: Strike amounts are used for strike prices in the terms 
- Collateral: Contains the original deposit and the payout.
- Quote: The priceAuthority's quotes are ERTP assets issued by a QuoteAuthority, which must be
  specified using the keyword Quote
```

Reply:
Honestly, I found the description of the four keywords contents to be very confusing. I'd suggest each one have two distinct parts; what is the value, and where is it coming from/how is it obtained. Definitely want to go with a list, as putting all four and explanations in a single paragraph is harder to follow.